.github/workflows/ci.yml

```diff
@@ -2332,3 +2332,46 @@ jobs:
           name: behat-logs-php${{ matrix.php }}-symfony${{ matrix.symfony }}
           path: build/logs/behat
         continue-on-error: true
+
+  windows-phpunit-mysql:
+    name: Windows PHPUnit + Behat (PHP 7.4) (SQLite)
+    runs-on: windows-latest
+    timeout-minutes: 10
+    env:
+      APP_ENV: sqlite
+      DATABASE_URL: sqlite:///%kernel.project_dir%/var/data.db
+      PGPASSWORD: root 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup PHP with pre-release PECL extension
+        uses: shivammathur/setup-php@v2
+        with:
+            php-version: '7.4'
+            tools: pecl, composer, flex
+            extensions: intl, openssl, mbstring, pdo_sqlite 
+            coverage: none
+            ini-values: memory_limit=-1
+      - name: Cache Composer packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.composer/cache
+          key: composer-php${{ matrix.php }}-${{ github.sha }}
+          restore-keys: |
+            composer-php${{ matrix.php }}-
+            composer-
+        continue-on-error: true
+      - name: Update project dependencies
+        run: |
+          composer update --no-progress --no-suggest --ansi
+      - name: Install PHPUnit
```

Comment:
This step is probably not necessary here (because we run PHPUnit before Behat anyway).